lte/gateway/c/core/oai/include/grpc_service.h

```diff
@@ -20,14 +20,16 @@
 #include "lte/gateway/c/core/oai/lib/bstr/bstrlib.h"
 #include "lte/gateway/c/core/oai/common/mme_default_values.h"
 
+#define TEST_GRPCSERVICES_SERVER_ADDRESS "127.0.0.1:50095"
```

Comment:
why is this defined here but not part of the test suite?

Reply:
@rsarwad

Reply:
grpc_service_init() was declared in this file. TEST_GRPCSERVICES_SERVER_ADDRESS is passed as an argument. So I declared it here.

Reply:
@rsarwad , to @ulaskozat's point, you can move this to `mock_tasks.h` and still use it in the call to `grpc_service_init()` in `sgw_s8_recv_grpc_messages.cpp`

Reply:
Addressed the comment in PR, https://github.com/magma/magma/pull/10370